setup.cfg

```diff
@@ -138,6 +139,32 @@ all =
     paramiko
     snowflake-sqlalchemy>=1.4.4  # Temporary solution for https://github.com/astronomer/astronomer-providers/issues/958, we should pin apache-airflow-providers-snowflake version after it pins this package to great than or equal to 1.4.4.
 
+# Hive provider does not work for python-3-11
+# See GihHub issue https://github.com/cloudera/python-sasl/issues/30
+# So creating this target for testing
+test_python_3_11 =
```

Comment:
We have a pre-commit [hook](https://github.com/astronomer/astronomer-providers/blob/main/.circleci/scripts/pre_commit_setup_cfg_all_extra.py) to ensure that the extra `all` contains all the dependencies listed in other extras. Can we add a similar pre-commit hook to ensure that `test_python_3_11` also contains all dependencies apart form `Hive` provider so that we do not miss adding newer dependencies here?

Reply:
I have added one PTAL.